Examples/Test_SLIT.py

```diff
@@ -75,17 +76,18 @@ def SIE(x0,y0,n1,n2,b,beta,q,xc,theta):
 ################################Running SLIT############################
 #Parameters
 kmax = 5
-niter =100
+niter = 100
 levels = [0]
 
 #Comment the following to have the level estimation routine run (takes more time)
-levels = pf.open('../Files/Noise_levels_SLIT.fits')[0].data
+# levels = pf.open('../Files/Noise_levels_SLIT.fits')[0].data
 
 #Start clock
 start = time.clock()
 
 #Running SLIT
-sourcesl, Imsl = SLIT.SLIT(Image, Fkappa, kmax, niter, size, PSF, PSFconj, levels = levels, fb =1)
+sourcesl, Imsl, Res1, Res2, sigma0 = SLIT.Solve.SLIT(Image, Fkappa, kmax, niter, size, PSF, PSFconj, 
+                                                     levels=levels, scheme='FB', show_plots=True)
```

Comment:
PSFconj should go. Indeed, it just determined by the transpose of the PSF itself and it's already in the code.